Packs/PrismaSaasSecurity/Integrations/SaasSecurityEventCollector/SaasSecurityEventCollector_description.md

```diff
@@ -1,11 +1,13 @@
 
 ## Limitations
 1) Occurring events expire after one hour in Saas-Security cache, so setting a low limit could cause events to expire if there are a large number of events in the Saas-Security cache.
-2) The max-fetch/limit parameters to fetch events must be divisible by 100.
+2) The max-fetch/limit parameters to fetch events must be divisible by 10.
```

Comment:
I would change this to a disclaimer that we will round down (lets care for the 0 case)